### PR TITLE
Document loading inputs configuration from external files in standalone mode

### PR DESCRIPTION
**Pull request by kvch**
In https://github.com/elastic/beats/pull/30087 we added support for loading input configuration from external configuration files in Elastic Agent when running in standalone mode. The behaviour is similar to Filebeat's support. However, at the moment path to the inputs folder is not configurable.

This PR adds a few sections to the standalone agent configuration page.

### CODE REVIEW

File: docs/en/ingest-management/elastic-agent/configuration/elastic-agent-configuration.asciidoc [hunk 1]
```diff
@@ -21,6 +21,32 @@ changes after installation, you must modify the installed file.
 
 For installation details, refer to <<install-standalone-elastic-agent>>.
 
+Alternatively, you can put input configurations in YAML files into the
+folder `{path.config}/inputs.d` to separate your configuration into
+multiple smaller files.
+Glob that has to match the path is `{path.config}/inputs.d/*.yml`.
```
Comment (dedemorton) [inline docs/en/ingest-management/elastic-agent/configuration/elastic-agent-configuration.asciidoc:27]: Is there something missing here? This sentence doesn't make sense in the context of this paragraph.

File: docs/en/ingest-management/elastic-agent/configuration/elastic-agent-configuration.asciidoc [hunk 2]
```diff
@@ -21,6 +21,32 @@ changes after installation, you must modify the installed file.
...
+From external configuration files ONLY input configurations are loaded.
```
Comment (dedemorton) [inline docs/en/ingest-management/elastic-agent/configuration/elastic-agent-configuration.asciidoc:28]: I think the word order here makes this sentence a bit harder to parse. 

Maybe say something like this instead: "The YAML files in the `inputs.d` folder should contain input configurations only. Any other configurations are ignored."

File: docs/en/ingest-management/elastic-agent/configuration/elastic-agent-configuration.asciidoc (hunk 1 above)
Comment (dedemorton) [reply docs/en/ingest-management/elastic-agent/configuration/elastic-agent-configuration.asciidoc:27]: Do users need to configure settings in the elastic-agent.yml file to enable use of the inputs.d folder?

**Review by dedemorton — COMMENTED** [review]
Thanks for doing this!

File: docs/en/ingest-management/elastic-agent/configuration/elastic-agent-configuration.asciidoc (hunk 1 above)
Comment (kvch) [reply docs/en/ingest-management/elastic-agent/configuration/elastic-agent-configuration.asciidoc:27]: It was just a leftover from one of my drafts. Thanks.

> Do users need to configure settings in the elastic-agent.yml file to enable use of the inputs.d folder?

 No, Agent automatically picks up files from the inputs.d folder.

**Review by dedemorton — APPROVED** [review]
Minor suggestion. Otherwise LGTM. Sorry I lost track of this one.

**Pull request merged**